src/inputs/checkbox.jsx

```diff
@@ -133,34 +197,43 @@ class Checkbox extends Component {
         const inputId = id ? `${id}_hidden_input` : null;
 
         return (
+            // eslint-disable-next-line jsx-a11y/no-static-element-interactions
             <div
                 className={containerClasses}
                 id={id}
                 onClick={this.onClick}
+                onKeyDown={this.onKeyDown}
                 style={style}
+                tabIndex={tabIndex}
             >
                 <input
                     checked={isChecked}
                     className="input"
-                    disabled={newDisabled}
+                    disabled={isDisabled}
                     id={inputId}
                     name={name}
                     readOnly
+                    // eslint-disable-next-line no-underscore-dangle
                     ref={(ref) => { this._inputRef = ref; }}
                     type="checkbox"
                     value={newValue}
                 />
 
-                <label className={labelClasses}>
-                    {label ? (
+                {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+                <label
+                    className={labelClasses}
+                >
```

Comment:
Instead of suppressing the warning, couldn't we just do `htmlFor={inputId}`?

Reply:
I like it... fixing.

Reply:
Commit 7c8b691 fixes this.

Reply:
Looks good.  Resolving.